src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1497,6 +1497,24 @@ def use_mutator_plugin(target_name, extra_env):
   return True
 
 
+def use_radamsa_mutator_plugin(extra_env):
+  """Decide whether to use Radamsa in process. If yes, add the path to the
+  radamsa shared object to LD_PRELOAD in |extra_env| and return True."""
+
+  # Radamsa will only work on LINUX ASAN jobs.
+  # TODO(mpherman) Include Architecture Info in Job Definition and exclude i386.
```

Comment:
I think we usually put a colon after the paren. 
Also this sentence has a weird amount of capitalized words.